migrazioni/Max_2_Magazzino.sql

```diff
@@ -0,0 +1,180 @@
+-- phpMyAdmin SQL Dump
```

Comment:
le migrazioni non sono dei dump del database. In ogni migrazione dovrebbe esserci solo quello che cambia rispetto alla precedente